### PR TITLE
Add Debian9 Dockerfile

### PR DESCRIPTION
**Pull request by ianhays**
cc: @MichaelSimons @janvorli

### CODE REVIEW

File: src/debian/stretch/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            libicu-dev \
+            libkrb5-dev \
+            liblldb-3.9-dev \
+            liblttng-ust-dev \
+            libssl1.0-dev \
+            libssl1.0.2 \
+            libunwind8-dev \
+            lldb-3.9 \
+            llvm \
+            make \
```
Comment (janvorli) [inline src/debian/stretch/Dockerfile]: A nit- the make is listed twice here.

File: src/debian/stretch/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    && apt-get install -y \
+            clang \
+            cmake \
+            curl \
+            g++ \
+            gettext \
+            git \
+            gnupg \
+            libcurl4-openssl-dev \
+            libgdiplus \
```
Comment (janvorli) [inline src/debian/stretch/Dockerfile]: What is the libgdiplus needed for?

Comment (ianhays) [reply src/debian/stretch/Dockerfile]: I'm not sure, but it's in most of our Dockerfiles e.g. [Debian8.2](https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/master/src/debian/8.2/coredeps/Dockerfile#L23)

File: src/debian/stretch/Dockerfile (hunk 1 above)
Comment (ianhays) [reply src/debian/stretch/Dockerfile]: thanks, fixed.

**Comment by MichaelSimons** [comment]
@ianhays - I just realized the new Dockerfile wasn't added to the [manifest](https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/master/manifest.json).  The manifest instructs the system what to build and how it should be tagged.  Can you please create a PR to update the manifest?

**Pull request merged**